package.json

```diff
@@ -27,15 +27,18 @@
     "precompile": "gts clean"
   },
   "dependencies": {
+    "@google-cloud/paginator": "^5.0.0",
+    "apache-arrow": "^14.0.2",
+    "core-js": "^3.37.1",
```

Comment:
gax handles paginated calls here - https://github.com/googleapis/gax-nodejs/tree/main/gax/src/paginationCalls but these would typically be generated. I suspect that they're not relevant here and since the package is small and this is mainly around typing, I'll resolve this